docs/documentation/aggregates/common.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Common Aggregates
+---
+
+<Note>
+  Please read the [overview](/documentation/aggregates/overview) for important information on how to accelerate aggregate queries.
+</Note>
+
+## Count
+
+`COUNT` tallies the number of values for the specified field across all documents.
```

Comment:
```suggestion
`COUNT` tallies the number of matching rows/documents from the table.
```

A query that does what you say here would be:

```sql
SELECT COUNT(DISTINCT fast_field) FROM mock_items WHERE description @@@ 'shoes';
```
... which I see you have below as "Cardinality".